frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/MenuStrategyRemove/DisableEnableStrategyDialog/DisableEnableStrategyDialog.tsx

```diff
@@ -28,7 +30,7 @@ export const DisableEnableStrategyDialog = ({
             if (disabled) {
                 onSuggestEnable();
             } else {
-                onSuggestEnable();
+                onSuggestDisable();
```

Comment:
I'd add a test for The DisableEnableStrategyDialog replicating a failing scenario. I see the DisableEnableStrategyDialog doesn't have any tests.

Reply:
I was doing similar component recently and ended up writing this test so you may use it as source of inspiration: https://github.com/Unleash/unleash/pull/3803/files#diff-519bb30b2e3d5f3c128a52eed4952c3c64bb5d945ac785d112c819f52892e0e3

Reply:
Will do